pkg/vm/memory/relocatable.go

```diff
@@ -20,16 +22,14 @@ func NewRelocatable(segment_idx int, offset uint) Relocatable {
 	return Relocatable{segment_idx, offset}
 }
 
-func (r *Relocatable) RelocateAddress(relocationTable *[]uint) uint {
-	return (*relocationTable)[r.SegmentIndex] + r.Offset
+func (r *Relocatable) RelocateAddress(relocationTable *[]uint) lambdaworks.Felt {
+	return lambdaworks.FeltFromUint64(uint64((*relocationTable)[r.SegmentIndex] + r.Offset))
 }
 
 // Int in the Cairo VM represents a value in memory that
 // is not an address.
 type Int struct {
-	// FIXME: Here we should use Lambdaworks felt, just mocking
-	// this for now.
-	Felt uint
+	Felt lambdaworks.Felt
 }
```

Comment:
If we are using `Felt` directly we should remove this struct to avoid confusions

Reply:
Done